### PR TITLE
feat: show Deploy in the header with 1% probability

### PR DESCRIPTION
**Pull request by satyarohith**
This PR also re-positions the install link as the main call to action and moves the version link to the bottom of the install button.

With "Deploy" in the header:
CleanShot 2021-08-18 at 11 33 06@2x

Without "Deploy" in the header:
CleanShot 2021-08-18 at 11 27 41@2x

---

The failing test of the PR is unrelated to the changes.

### CODE REVIEW

File: components/Header.tsx [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
+ * should be displayed to the user.
+ */
+function displayDeploy() {
+  const showDeploy = localStorage.getItem("showDeploy");
+  if (showDeploy !== null) {
+    return showDeploy === "true";
+  } else {
+    // 1% probability
+    const showDeploy = Math.random() < 1 / 100;
+    localStorage.setItem("showDeploy", String(showDeploy));
```
Comment (ry) [inline components/Header.tsx:18]: it seems like over time, if people keep visiting the site they will eventually be pushed into the "showDeploy" category?

Comment (satyarohith) [reply components/Header.tsx:18]: The else block should only run on the first visit unless the "showDeploy" key is cleared from localStorage.

File: components/Header.tsx [hunk 2]
```diff
@@ -12,6 +28,8 @@ function Header({
   widerContent?: boolean;
 }): React.ReactElement {
   const [menuOpen, setMenuOpen] = useState(false);
+  const [showDeploy, setShowDeploy] = useState(false);
+  useEffect(() => setShowDeploy(displayDeploy()), []);
```
Comment (kt3k) [inline components/Header.tsx:32]: Seems like it's simpler to use `useMemo`

```suggestion
  const showDeploy = useMemo(() => displayDeploy(), []);
```

Comment (satyarohith) [reply components/Header.tsx:32]: `useMemo()` will also run on the server.

Comment (kt3k) [reply components/Header.tsx:32]: Ah, ok. makes sense!

**Review by ry — APPROVED** [review]
LGTM - assuming you can get CI green

**Pull request merged**